### DISCUSSION TITLE
感想みたいなもの

### DISCUSSION THREAD

**Discussion by minatani** [Umio-Yasuno/umio-yasuno.github.io#6] [Show and tell]
いつも興味深く拝見しています。以下のような募集を見つけました。ゲーム開発扱いですが、GPU寄りのようなのでお知らせしてみます。
日本 東京でAMDがSoftware Development Eng. - Game Programmer Engineerを募集中 | LinkedIn https://www.linkedin.com/jobs/view/2632845928/
まあそれはそれとして、以前「こんなサイト誰が見てるんだ？」と書いていらっしゃった記憶がありますが、楽しみにしている人間もいるという感想でした。

**Comment by Umio-Yasuno** [comment]
感想ありがとうございます。
直接感想をもらったのはこれが初めてなので純粋に嬉しいです。

自分はやっぱり眺めているだけのオタクで、GPU向けのコードを書ける人間ではないので残念ですがその募集には向いてなさそうです。
それでも海のどこかでゆっくりと泳いでいるシーラカンスにように、オタクなりにひっそりと頑張ってサイトを更新していくつもりでいます。
